src/page-objects/administration/Categories.ts

```diff
@@ -51,8 +49,7 @@ export class Categories implements PageObject {
     this.addLandingPageButton = this.landingPageArea.getByText('Add landing page');
     this.landingPageItems = this.landingPageArea.locator('.sw-tree-item__label');
     this.categoryTree = page.locator('.sw-category-tree');
-    this.homeCategory = this.categoryTree.locator('.sw-tree-item__element').filter({ hasText: 'Home' });
```

Comment:
if we remove that, we have to adjust the a platform test or?

Reply:
@vanpham-sw you aware of it?

Reply:
Yes, we have to adjust the platform test:
From this:
`await ShopAdmin.attemptsTo(CreateLinkTypeCategory(categoryData, categoryCustomizableLinkData));`
to:
`await ShopAdmin.attemptsTo(CreateLinkTypeCategory(categoryData, categoryCustomizableLinkData, 'Home'));`

I can create a MR, but maybe there is a better way?

Reply:
I plan to do that :)